File: Php/app/Urls/CashgameDetailsChartJsonUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\CashgameUrlModel;
	use app\UrlFormatter;
	use entities\Cashgame;
	use entities\Homegame;

	class CashgameDetailsChartJsonUrlModel extends CashgameUrlModel{

		public function __construct(Homegame $homegame, Cashgame $cashgame){
			parent::__construct(RouteFormats::cashgameDetailsChartJson, $homegame, $cashgame);
		}

	}

}

File: Web/Models/UrlModels/CashgameDetailsChartJsonUrlModel.cs
using Core.Classes;
using Web.Routing;

namespace Web.Models.UrlModels{

	public class CashgameDetailsChartJsonUrlModel : CashgameUrlModel{

		public CashgameDetailsChartJsonUrlModel(Homegame homegame, Cashgame cashgame)
            : base(RouteFormats.CashgameDetailsChartJson, homegame, cashgame)
        {
		}

	}

}